### ISSUE TITLE
quiz: tensorflow dropout > quiz 2

### ISSUE THREAD

**Issue by mdesksmartin** [udacity/sdc-issue-reports#324]
instructions say "Apply dropout to the code from the ReLU Quiz. Use the keep_prob placeholder to pass in a probability of 0.5."  the ReLU quiz is in another earlier section.  is the reader supposed to figure out where the "relu quiz" was and go back to it and then come back to this page and scroll down to the bottom? if this quiz isn't reinforcing anything about "relu", there isn't a reason to refer to the previous code block.  

also, the comment in the quiz page of the code just says "print session results".  i would have expected the code to be a variation on the code example directly above which has the optimizer defined.  i'm not sure how someone would know that the output here is supposed to just be the logits.

**Comment by Brok-Bucholtz** [comment]
@mdesksmartin,

I reworded the sentence in Quiz 2 to better reflect what the ReLU Quiz has to do with this quiz.  I also clarified what I we want printed in the quiz.  Thanks for all your feedback :D
